docs/tutorials/google/start.ipynb

```diff
@@ -163,14 +163,24 @@
     }
    ],
    "source": [
+    "import os\n",
+    "\n",
     "# The Google Cloud Project id to use.\n",
     "project_id = \"\" #@param {type:\"string\"}\n",
-    "processor_id = \"pacific\" #@param {type:\"string\"}\n",
+    "processor_id = \"\" #@param {type:\"string\"}\n",
+    "\n",
+    "os.environ['GOOGLE_CLOUD_PROJECT'] = project_id\n",
     "\n",
     "from cirq_google.engine.qcs_notebook import get_qcs_objects_for_notebook\n",
     "device_sampler = get_qcs_objects_for_notebook(project_id, processor_id)\n",
     "\n",
-    "if not device_sampler.signed_in:\n",
+    "if device_sampler.signed_in:\n",
+    "  # Defaults to a processor if no processor_id is specified.\n",
+    "  if processor_id == '':\n",
+    "    processors = cg.get_engine(project_id).list_processors()\n",
+    "    if len(processors) > 0:\n",
+    "      processor_id = processors[0].processor_id\n",
```

Comment:
This should not be needed, because if we end up signed in the returned sampler will be a `QuantumEngineSampler` which stores the processor id, so you could do something like:
```python
obj = get_qcs_objects_for_notebook(project_id, processor_id)
if obj.signed_in:
    processor_id = obj.sampler._processor_ids[0]
```
Similarly, can get the project id from the sampler in this case:
```python
if obj.signed_in:
    project_id = obj.sampler._engine.project_id
```
We should probably expose these in a better way so that we don't need to access internal fields like this.

Reply:
It now looks like this should be `obj.processor_id`. However, the default processor is just the first one in the list, which may not be active.  For this quickstart, we should probably ensure that we pick a processor from the list `processor.health` is `OK`.